scheduler/graphql_mid/types.py

```diff
@@ -198,33 +206,13 @@ class SourceFileHandlerResponse:
 IQ = strawberry.enum(ImageQuality)
 
 
-@strawberry.type
-class NewWeatherChange:
-    start: datetime
-    reason: str
-    new_CC: Optional[CC]
-    new_SB: Optional[SB]
-    new_WV: Optional[WV]
-    new_IQ: Optional[IQ]
-
-    def to_scheduler_event(self) -> WeatherChange:
-        c = Conditions(cc=CloudCover[self.new_CC.name] if self.new_CC else None,
-                       sb=SkyBackground[self.new_SB.name] if self.new_SB else None,
-                       wv=WaterVapor[self.new_WV.name] if self.new_WV else None,
-                       iq=ImageQuality[self.new_IQ.name] if self.new_IQ else None)
-        return WeatherChange(start=self.start,
-                             reason=self.reason,
-                             new_conditions=c
-                             )
-
-
 @strawberry.type
 class EventsAddedSuccess:
```

Comment:
I'm thinking we might want to include the `uuid` in the `EventsAddedSuccess` so we can easily refer to an event by its `uuid`.